src/NuGet.Clients/NuGet.SolutionRestoreManager/VSNominationUtilities.cs

```diff
@@ -712,5 +712,15 @@ private static string[] HandleClear(string[] input)
 
             return input;
         }
+
+        internal static string GetSdkAnalysisLevel(IReadOnlyList<IVsTargetFrameworkInfo4> targetFrameworks)
+        {
+            return GetSingleNonEvaluatedPropertyOrNull(targetFrameworks, nameof(ProjectBuildProperties.SdkAnalysisLevel), v => v) ?? string.Empty;
+        }
+
+        internal static bool GetUsingMicrosoftNETSdk(IReadOnlyList<IVsTargetFrameworkInfo4> targetFrameworks)
+        {
+            return string.Equals(GetSingleNonEvaluatedPropertyOrNull(targetFrameworks, nameof(ProjectBuildProperties.UsingMicrosoftNETSdk), v => v), "true", StringComparison.OrdinalIgnoreCase);
```

Comment:
The `bool` type has `Parse` and `TryParse` methods. I'd use that, rather than string.Equals directly.